src/test/java/org/saltyrtc/client/tests/signaling/CloseCodeTest.java

```diff
@@ -0,0 +1,36 @@
+package org.saltyrtc.client.tests.signaling;
+
+import org.junit.Test;
+import org.saltyrtc.client.signaling.CloseCode;
+
+import java.lang.reflect.Field;
+import java.util.ArrayList;
+import java.util.List;
+
+import static org.junit.Assert.assertEquals;
+
+public class CloseCodeTest {
+
+    @Test
+    public void testAllCloseCodesInCLOSE_CODES_ALL() throws NoSuchFieldException, IllegalAccessException {
+        Field[] fields = CloseCode.class.getFields();
+
+        List<Integer> allCloseCodes = new ArrayList<>();
+        List<Integer> allCloseCodesInCLOSE_CODES_ALL = new ArrayList<>();
+
+        for (Field field : fields) {
+            if (field.getType().equals(int.class)) {
+                allCloseCodes.add(field.getInt(null));
+            }
+        }
+
+        Field CLOSE_CODES_ALL = CloseCode.class.getField("CLOSE_CODES_ALL");
+        int[] closeCodesInCLOSE_CODES_ALL = (int[]) CLOSE_CODES_ALL.get(null);
```

Comment:
I don't quite understand what you're doing here. Wouldn't `Assert.assertArrayEquals(allCloseCodes, CLOSE_CODES_ALL)` work?

Reply:
You're right. Using `CloseCode.CLOSE_CODES_ALL` directly is the way to go.
However, I have to transform it into a List. Otherwise I can't compare it.